### ISSUE TITLE
Capistrano v2 task no longer honors airbrake_env

### ISSUE THREAD

**Issue by cgriego** [airbrake/airbrake#777]
I'm trying to upgrade from a few pre v5 versions to the latest version, but the Capistrano 2 task no longer honors the airbrake_env Capistrano variable, even though the v3 task does. We run multiple 'staging' environments with the same framework (Rails) environment but are independent. We used the airbrake_env to keep deployment notifications separate.

https://github.com/airbrake/airbrake/blob/c91a7eee848c54dfe38a474b1425a2df99c58b1a/lib/airbrake/capistrano/tasks.rb#L48

**Comment by kyrylo** [comment]
Thanks for pointing out that inconsistency. I'm leaning towards removing `airbrake_env` from the capistrano3 integration. That said, I'm trying to understand why your Rails environment can't be configured to different stagings. It doesn't sound right that you rely on that variable to send some notifications. Is there any way you can make `rails_env` work for you?

**Comment by cgriego** [comment]
We run multiple server environments using the same application (Rails) environment, particularly "staging". We have a primary "staging" server environment used for our QA team, but we have additional sandbox server environments (named "sandbox1", "sandbox2", and so on) that use the same configuration as the "staging" server environment and they all use the "staging" application environment and configuration, but they are distinct servers running different versions of our code and when notifications already get reported to the right Airbrake environment using the server environment name ("staging1") by overriding Airbrake.environment, but without this fix then deployments to those environments would get reported to the wrong Airbrake environment.

**Comment by kyrylo** [comment]
There's no global state anymore and you can no longer override `Airbrake.environment` because it doesn't exist. Instead, you configure a notifier and set its `environment` option (typically `Rails.env`). 

So what do you do now? Do you have `config/deploy/sandbox1.rb`, `config/deploy/sandbox2.rb` ... `config/deploy/sandboxN.rb` and set `airbrake_env`?

```
set :airbrake_env, 'sandboxN'
```

Where do you define `airbrake_env`?

**Comment by cgriego** [comment]
You're right, `Airbrake.environment` doesn't exist anymore and I referenced it merely because we've only just recently upgraded to v5+ and that's why we ran into this issue and I filed this ticket. We are instead using `Airbrake.configure { |c| c.environment = AppConfig.environment }`, which will use the server environment like "sandbox1".

We use `set :airbrake_env, defer { fetch(:stage) }` in `config/deploy.rb` so it will use the stage identifier which matches the server environment, also "sandbox1". `Rails.env` is "staging" and we use `set :rails_env, "staging"` in the deployment config.
